Code/Tools/AssetProcessor/native/connection/connection.h

```diff
@@ -187,6 +187,7 @@ class Connection
     void DisplayNameChanged();
     void ElapsedChanged();
     void NormalConnectionRequested(QString IpAddress, quint16 Port);
+    void ConnectionReady(unsigned int connId, Connection* connection);
```

Comment:
This could take in a **Connection&** instead of a naked pointer, right?

Reply:
Updated to just pass the platforms instead, I saw an issue while debugging with the connection being disconnected (probably timed out due to taking too long in debug), this'll avoid any sort of use-after-free errors